packages/wallet/package.json

```diff
@@ -85,6 +86,8 @@
     "@cardano-sdk/tx-construction": "workspace:~",
     "@cardano-sdk/util": "workspace:~",
     "@cardano-sdk/util-rxjs": "workspace:~",
+    "@koralabs/handles-public-api-interfaces": "1.6.6",
+    "axios": "^1.4.0",
```

Comment:
This should be under `dependencies` since it's used in `src/`

Reply:
[f293dc8313](https://github.com/input-output-hk/cardano-js-sdk/pull/732/commits/f293dc8313a0c07bc570e18d025412315abc45ef) should solve this